skimage/measure/_colocalization.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
+from scipy.stats import pearsonr
+from functools import wraps
+from .._shared.utils import check_shape_equality, is_binary_ndarray
+
+__all__ = ['pearson_corr_coeff',
+           'manders_coloc_coeff',
+           'manders_overlap_coeff',
+           'intersection_coeff',
+           ]
+
+
+def pearson_corr_coeff(image0, image1, roi=None):
```

Comment:
I think `roi` should be renamed to `mask` to match other parts of the API?

Reply:
Yeah but could this then be confusing for functions that use only masks like `intersection_coeff(image0_mask, image1_mask, roi=None)`? Does naming it `mask` provide enough of a distinction against those other `image0_mask` and `image1_mask` variables?

Reply:
imho, yes, it does. :stuck_out_tongue_winking_eye: